core/jobs/batch_jobs/manual_voice_artist_name_job_test.py

```diff
@@ -57,9 +57,9 @@ class VoiceArtistMetadataModelsTestsBaseClass(
     EDITOR_USERNAME_3 = 'editor3'
     EDITOR_USERNAME_4 = 'editor4'
 
-    CURATED_EXPLORATION_ID_1 = 'exploration_id_1'
-    CURATED_EXPLORATION_ID_2 = 'exploration_id_2'
-    NON_CURATED_EXPLORATION_ID_3 = 'exploration_id_3'
+    CURATED_EXPLORATION_ID_1 = 'exp_id_abcde'
+    CURATED_EXPLORATION_ID_2 = 'exp_id_kjsd-'
```

Comment:
Added exploration ID with a hyphen at the end to mimic the real exploration ID **umPkwp0L1M0-**.